src/app/shared/pipes/translation-known-values.pipe.ts

```diff
@@ -0,0 +1,34 @@
+import { Pipe, PipeTransform } from '@angular/core'
+import { Tv2ActionContentType } from '../models/tv2-action'
+
+@Pipe({
+  name: 'translationKnownValues',
+})
+export class TranslationKnownValuesPipe implements PipeTransform {
```

Comment:
What constitues a `known value`? That's an extremely broad definition.
This pipe translate `Tv2ActionContentType` so a more accurate name would be `TranslateContentType`.